examples/shared_samples/server_hardware.rb

```diff
@@ -19,14 +19,16 @@
 #   @server_hardware_password
 #
 # Supported APIs:
-# - 200, 300, 500
+# - 200, 300, 500, 600
 
 # Resources that can be created according to parameters:
 # api_version = 200 & variant = any to OneviewSDK::API200::ServerHardware
 # api_version = 300 & variant = C7000 to OneviewSDK::API300::C7000::ServerHardware
 # api_version = 300 & variant = Synergy to OneviewSDK::API300::Synergy::ServerHardware
 # api_version = 500 & variant = C7000 to OneviewSDK::API500::C7000::ServerHardware
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::Synergy::ServerHardware
+# api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::ServerHardware
```

Comment:
for PATCH, updating scopes has been removed, so add condition for API600

Reply:
PATCH endpoint is not removed for this resource.